src/main/webapp/view/discussion/discussion.js

```diff
@@ -128,6 +137,16 @@ async function fetchDiscussion() {
   return request.json();
 }
 
+/**
+ * Updates the timestamp displayed and sent by the new comment form.
+ *
+ * @param {number} timeMs The new time in milliseconds to use.
+ */
+function updateNewCommentTimestamp(timeMs) {
+  ELEMENT_TIMESTAMP_SPAN.innerText = window.timestampToString(timeMs);
+  newCommentTimestampMs = timeMs;
```

Comment:
What about currentVideoTimeMs as a variable name instead of newCommentTimestampMs. Although this timestamp is used for the new comment timestamp it really is a representation of the video time.

Reply:
I'm worried about name conflicts because that name is kinda generic and in the global scope...

Reply:
I added a TODO to bring this into a class, module, or namespace.  See #191